entrypoint.sh

```
@@ -2,6 +2,28 @@
 
 # Terminate upon errors
 set -e
+
 clang-format --version
-# Run format script
-python3 /run-clang-format.py "$@"
+
+# Parse arguments
+# Called from clangFormatCheck.sh as: entrypoint.sh <scandir> <excludedirs> <extensions> <style> [--fix]
+SCANDIR="${1:-render-node/src}"
+EXCLUDEDIRS="${2:-}"
```

Shellcheck (0.11.0)

[warning] EXCLUDEDIRS appears unused. Verify use (or export if used externally).

(SC2034)